manifests/charts/cloudcore/values.yaml

```diff
@@ -99,3 +99,32 @@ iptablesManager:
     requests:
       cpu: 100m
       memory: 25Mi
+
+controllerManager:
+  enable: false
+  hostNetWork: true
+  image:
```

Comment:
Is this component needed host network? I'm not familiar with it.

Reply:
no, these deployment files are inherit from dir https://github.com/kubeedge/kubeedge/tree/master/build/controllermanager

Reply:
it does not need host network